### PR TITLE
Created 'Activate Created Forecasts' test case and created few new methods and locators. 

### PR DESCRIPTION
**Pull request by neelimaboyina4018**
Created 'Activate Created Forecasts' test case and created few new methods and locators.

### CODE REVIEW

File: src/test/java/step_definitions/salesforceDemo.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public void read_the_test_data_from_Excel(String arg1) {
     @When("^navigate to salesforce$")
     public void navigate_to_the_url() throws Exception {
 
-        KeywordUtil.navigateToUrl("https://testing-f5-dev-ed.develop.my.salesforce.com");
+        KeywordUtil.navigateToUrl("https://testingxperts17-dev-ed.develop.my.salesforce.com/");//neelima SF URL
```
Comment (snaphray) [inline src/test/java/step_definitions/salesforceDemo.java]: Please don't remove the line to navigate to the url, just comment it while testing and while committing just comment your line and uncomment the original one

File: src/test/java/SalesforceModules/ForecastingModule.java [hunk 2]
```diff
@@ -184,4 +184,82 @@ public static void verifyRoleAssignToUSer() throws InterruptedException {
             catchAssertError(e);
         }
     }
-}
+    public static void validateForecastSettingsPageShouldBeLoaded() throws InterruptedException {
```
Comment (snaphray) [inline src/test/java/SalesforceModules/ForecastingModule.java:567]: Please add proper comments for each method

**Review by snaphray — CHANGES_REQUESTED** [review]
Please check review comments

**Comment by neelimaboyina4018** [comment]
Hi @snaphray, Updated files as per the review comments.

**Review by snaphray — APPROVED** [review]
Code looks fine now

**Pull request merged**